app/controllers/game.js

```diff
@@ -330,6 +330,16 @@ export default Ember.Controller.extend({
     },
 
     /**
+     * Tap or untap a card.
+     *
+     * @param {GameCard} gameCard
+     */
+    tap: function (gameCard) {
+      gameCard.toggleProperty('isTapped');
+      this.get('model').save();
```

Comment:
I appreciate you modifying the game card on the controller. Data down, actions up!